tests/ssl-opt.sh

```diff
@@ -2969,6 +2965,8 @@ run_test    "Max fragment length: client 512, server 1024" \
             -s "server hello, max_fragment_length extension" \
             -c "found max_fragment_length extension"
 
+requires_config_value_at_least "MBEDTLS_SSL_IN_CONTENT_LEN" 2048
+requires_config_value_at_least "MBEDTLS_SSL_OUT_CONTENT_LEN" 2048
```

Comment:
That's very curious, as I've successfully gotten that test to pass. Both IN and OUT are set to 2048 (confirmed also by echo-ing in the script). Would you be able to extract any information from the log?

Reply:
I can't reproduce the failure here either.

Reply:
Hm.. Yes, seems like this particular testcase stopped reproducing on my machine since yesterday.
Anyways, I launched whole ssl-opt.sh suite with modified mbedtls_config.h (rebased on development)
And got bunch of failures.
[git patch](https://github.com/ARMmbed/mbedtls/files/6753201/reproduce_failure.txt)
[logs](https://github.com/ARMmbed/mbedtls/files/6753196/logs.tar.gz)

Reply:
You're right, it seems there are quite a few test cases several thousand lines below these which implicitly depend on IN/OUT being above a certain threshold (example being those run with `-f "Large server packet"`).

It seems however that even on `development` (without this PR), those tests fail when IN/OUT is set to 4096 or even 8192. Should I still try to find out the conditions that make them fail, or should this be another issue?

Reply:
if this is preexisting then perhaps it should be fixed in new bug ticket.